test/OpenTelemetry.Instrumentation.AspNet.Tests/TestActivityProcessor.cs

```diff
@@ -0,0 +1,23 @@
+// <copyright file="TestActivityProcessor.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+
+namespace OpenTelemetry.Instrumentation.AspNet.Tests;
+
+internal class TestActivityProcessor : BaseProcessor<Activity>
```

Comment:
Could you reuse the [TestActivityProcessor ](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/test/OpenTelemetry.Contrib.Tests.Shared/TestActivityProcessor.cs) from `OpenTelemetry.Contrib.Tests.Shared` instead of creating a new one here?